Makefile

```diff
@@ -30,7 +30,7 @@ NPROCS ?= 1
 GO_TEST_PARALLEL := $(shell echo $$(( $(NPROCS) / 2 )))
 
 GO_STATIC_PACKAGES = $(GO_PROJECT)/cmd/crossplane $(GO_PROJECT)/cmd/crank
-GO_LDFLAGS += -X main.version=$(VERSION)
+GO_LDFLAGS += -X $(GO_PROJECT)/pkg/version.version=$(VERSION)
```

Comment:
I think this will break the Crossplane CLI

Reply:
This is just copied from master; why would it break?

Reply:
Ah, I see - because this change was made after v0.13. Will revert.